### ISSUE TITLE
Issues with installing nitroshare

### ISSUE THREAD

**Issue by shag00** [nitroshare/nitroshare-desktop#261]
Kubuntu 20.10

Install 0.3.1 from https://launchpad.net/nitroshare/0.3/0.3.1#release-data
package installer reports unmet dependencies

install using: sudo apt-get install nitroshare:
scott@scottubuntu:~$ sudo apt-get install nitroshare
[sudo] password for scott: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package nitroshare

I think someone needs to do a little bit of housekeeping on this issue.
`

**Comment by dinnerisserved** [comment]
I have same error on Ubuntu 20.04

**Comment by pranav083** [comment]
I am also having the same problem still not solved for `ubuntu 20.10`.

**Comment by reaper92345** [comment]
Same on kali

**Comment by mitchierichie** [comment]
Yeah on 20.04 I'm not able to build using cmake either, and if I add their PPA it says it's insecure or something.

**Comment by CoderBoogs** [comment]
Same issue on Pop os 21.10

**Comment by shamimlem** [comment]
└─# sudo apt-get install nitroshare

Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
_E: Unable to locate package nitroshare_


**I FACE SAME PROBLEM PLS HELP ME ANYBODY**

**Comment by fakhamatia** [comment]
Ubuntu 21.10 !

**Comment by LixvYang** [comment]
This will be ok!
```sh
sudo apt-add-repository ppa:george-edison55/nitroshare
sudo apt-get update
sudo apt-get install nitroshare
```

**Comment by bajurski** [comment]
Unfortunately it doesn't. On my Deb 11 the problem remains.
